android/gradle.properties

```diff
@@ -23,4 +23,4 @@ org.gradle.daemon=true
 org.gradle.jvmargs=-Xmx2560m
 
 # Version of flipper SDK to use with React Native
-FLIPPER_VERSION=0.99.0
\ No newline at end of file
+FLIPPER_VERSION=0.93.0
```

Comment:
might be a stupid question, but downgraded version is correct one for upgraded RN?

Reply:
I mistakely upgraded to higher version then required and it didn't work ;) So this version - correct

Reply:
Ok